### ISSUE TITLE
Website Request

### ISSUE THREAD

**Issue by Tetdesu** [mHaisham/novelsave#36]
Can I request this site?
https://rainofsnow.com/mr-dior/#chapter

:3

**Comment by mHaisham** [comment]
Certainly. Unfortunately though, I am currently in the process of re-writing the application #35, so you might have to wait a bit. Most of the existing functionality are implemented, so shouldn't be long now.

P.S. Sources have been moved to a new repository [`novelsave_sources`](https://github.com/mHaisham/novelsave_sources).

**Comment by Tetdesu** [comment]
Its ok I can wait. Take your time and Thank you ☺️

**Comment by mHaisham** [comment]
If you dont mind a bug here and there and some missing features, feel free to use the [alpha version](https://pypi.org/project/novelsave/0.7.0a2/) (The README is yet to be updated 😅).

```
pip install novelsave==0.7.0a2
```

the commands are as follows:

```
novelsave update <url>
novelsave compile <url>
novelsave refresh <url>
```

`refresh` runs update and then compile consecutively

So in this case the following command should suffice.

```
novelsave refresh https://rainofsnow.com/mr-dior/
```

You'll find the epubs in `~/novels` under a folder named after the source (may change before full release).

P.S. I would appreciate you sharing any bugs you may find greatly.

**Comment by Tetdesu** [comment]
Its ok. And sure ill notify you if i do find some bugs. Thank you so much 😘
